src/components/page/Page.tsx

```diff
@@ -0,0 +1,17 @@
+import './page.scss';
+
+type propsType = {
+  children?: JSX.Element
+}
+
+export default function Page(props: propsType) {
+
+  return (
+    <div className='page'>
+      <div className='page__container'>
+        {props.children && props.children}
```

Comment:
Не уверен, что здесь нужна проверка на наличие children. Я бы сделал так:

```suggestion
        {props.children}
```